apps/core/components/ItineraryCard/TripSectorHelpers.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+import * as DateFNS from 'date-fns';
+
+export const timeSimpleFormat = 'H:mm';
+export const dateFormat = 'ddd D MMM';
+const durationFormat = 'HH[h] m[m]';
+export const getFormattedDate = (
+  time: ?string,
+  format: ?string = timeSimpleFormat,
+) => DateFNS.format(DateFNS.parse(time), format);
+
+export const getDuration = (durationInMinutes: ?number) => {
+  if (!durationInMinutes) {
+    return null;
+  }
+  const date = new Date(0, 0, 0, 0, durationInMinutes);
+  return durationInMinutes && DateFNS.format(date, durationFormat);
```

Comment:
do you need durationInMinutoes here, you already checked in the if that it is not falsy

Reply:
You are right. Fixed